lang/en.js

```diff
@@ -33,6 +33,9 @@ export default {
 	"components.input-date.errorOutsideRange": "Date must be between {minDate} and {maxDate}",
 	"components.input-date.openInstructions": "Use date format {format}. Arrow down or press enter to access mini-calendar.",
 	"components.input-date.setToToday": "Set to Today",
+	"components.input-number.errorMaxNumberOnly": "{label} must be lower than {maxNumber}.",
+	"components.input-number.errorMinNumberOnly": "{label} must be higher than {minNumber}.",
```

Comment:
It'd recommend reusing the lang terms from `components.form-element.input.number.rangeOverflow` and `components.form-element.input.number.rangeUnderflow`